sensor_test.py

```diff
@@ -28,10 +32,10 @@ def test_sanity(get_sensor_info, get_sensor_reading):
         sensor_reading, float
     ), "Sensor doesn't seem to register temperature"
 
-    print("Sanity test passed")
+    log.info("Sanity test passed")
 
 
-def test_reboot(get_sensor_info, reboot_sensor):
+def test_reboot(get_sensor_info, sensor_reboot):
```

Comment:
You're getting a `fixture not found error` because fixture name here should be `reboot_sensor` instead of `sensor_reboot`.